cypress/support/config/settings.js

```diff
@@ -4217,8 +4217,11 @@ module.exports = () => ({
             enabled: false,
           },
           test: {
-            paths: [],
-            enabled: false,
+            paths: [
+              '/russian/av/media-52355574',
```

Comment:
It would make our lives easier if we just forgot this `/av` was ever a thing :)

It shouldn't exist anymore once we go live

The redirect is just there to deal with any links on other sites (eg, twitter, facebook) that is using the old URLs, but from the BBC's perspective,  `russian/av` URLs don't exist any more

Reply:
Ah interesting. With this in mind - when live - if we just fed in urls like '/russian/media-52355574' to the test, would we expect the usual test suite to just 'work' without any workarounds? I can't remember if the embed endpoint is also constructed differently for Russian?

Reply:
I suppose it is just to test that the redirect does work on those urls, even though they will be very rare. I can also add normal non /av/ urls, and will definitely in the live url PR tomorrow

Reply:
> Ah interesting. With this in mind - when live - if we just fed in urls like '/russian/media-52355574' to the test, would we expect the usual test suite to just 'work' without any workarounds? I can't remember if the embed endpoint is also constructed differently for Russian?

The test for the player, and all other tests, work normally on the TC2 URL. I expect it would work the same on a normal CPS one. We can find out!